docker/docker-python/build_python_dockerfile.sh

```diff
@@ -117,10 +131,40 @@ docker_build_image "${mxnet_version}_gpu_cu92_mkl" "Dockerfile.mxnet.python.gpu.
 docker_test_image_gpu "${mxnet_version}_gpu_cu92_mkl"
 
 
+
+# Build and Test Python3 dockerfiles - CPU
+docker_build_image "${mxnet_version}_cpu_python3" "Dockerfile.mxnet.python3.cpu"
```

Comment:
I'd prefer if the builds and CPU tests ran parallelized to reduce the critical path

Reply:
I'd be a little concerned about memory usage if we were to parallelize the builds.  I could see that this might make building the images difficult on a laptop or even a middle-tier cloud instance.

Reply:
I built 15 images for our docker cache in parallel on my laptop and didn't encounter any problems. I don't think it will be an issue. Also, this script is targeted for CD execution - individual compilation could be an additional mode, but we should optimise for maximum resource utilization here IMO